docs/styleguide.config.js

```diff
@@ -1,8 +1,11 @@
 const path = require('path')
 
 module.exports = {
-  components: path.resolve(__dirname, '../react/{Button,Toggle}/*.{js,jsx}'),
+  components: path.resolve(__dirname, '../react/{Spinner,Icon,Button,Toggle}/*.{js,jsx}'),
```

Comment:
We shall not list components here, if we have 30 000 components, we could not list them anymore, could we?

Reply:
Exactly what I was talking about here : https://github.com/cozy/cozy-ui/pull/180#pullrequestreview-62379744

I am doing the change

Reply:
done